### ISSUE TITLE
Question Regarding Loss Function

### ISSUE THREAD

**Issue by DannySalem** [aspuru-guzik-group/selfies#4]
Hello, I'm looking to extend your work to another application. We are planning to use Beta-VAE but I noticed you already have a constant alpha infront of the KLD; it's also very very low (0.0001).  Conventional VAE loss would have a 1 there, can you provide some intuition into what the function of the low KLD_alpha is for before I start playing around?

Thanks

**Comment by MarioKrenn6240** [comment]
Hi Danny, thanks for your question. In the test-code, I set it to an arbitrary number. In the actual experiment, we use KLD_alpha as a hyperparameter in the bayesian optimisation. We didn't see much influence of it's value in our experiments, and didn't investigate further its property. However, investigations for beta-VAEs, this value gets important, and I am curious about your results. Please let us know.
